### DISCUSSION TITLE
Dependence to python since --cpu-proof command

### DISCUSSION THREAD

**Discussion by totomakers** [chakra-ui/panda#2138] [General]
Since adding `https://panda-css.com/docs/references/cli#--cpu-prof`

@pandacss/core need to intall `v8-profiler-next` that go a deps to  node-pre-gyp
We now need install python along side for working panda working with pnpm (but i guess any other package manager) 

It's normal or i miss something in the doc ?

**Comment by astahmer** [comment]
I'll refactor that part to use the native node profiler instead of that `v8-profiler-next` module

**Reply by totomakers** [reply]
Perfect <3

 if anyone encounter the issue:

+ If you use Alpine for deploy your frontend:

```
RUN apk add --no-cache python3 make g++
```

+ On windows, install python following:
https://github.com/nodejs/node-gyp?tab=readme-ov-file#on-windows

**Reply by astahmer** [reply]
https://github.com/chakra-ui/panda/pull/2139 will probably be part of next release
